src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+@import "../utils/variables";
```

Comment:
You could add this to the top of main.scss

Reply:
Done.